eos-enable-zram

```diff
@@ -54,6 +54,17 @@ if [ -z "${root_disk}" ]; then
     exit 0
 fi
 
+for swap_area in "${active_swaps[@]}" ; do
+    if [ "${swap_area%?}" = "${root_disk}" ] ; then
+        echo "Found swap partition ${swap_area} on root disk"
+	found=1
```

Comment:
mismatched indentation

Reply:
Good eye, and a good reminder for me to turn my tab highlighter back on!